xfce-base/thunar/thunar-4.19.2.ebuild

```diff
@@ -50,6 +50,10 @@ BDEPEND="
 	virtual/pkgconfig
 "
 
+PATCHES=(
+	"${FILESDIR}/thunar-gio-extensions.c.patch"
```

Comment:
Make it `${P}-build.patch` or sth like that. Also, please add a comment here, linking to upstream commit.